DOCUMENTATION.md

```diff
@@ -0,0 +1,7 @@
+# Documentation of the changes made
+
+In the refactoring of this code, the first thing I noticed was the fact that there was no protected module. The protected module keeps methods from being mistakenly called. So, the first thing I did was to add the protected module. I also noticed that the articleParams method was improperly named so I renamed it article_params everywhere it was called.
```

Comment:
What does protected module mean? I think you were referring to protected methods?

Reply:
Can you say why it needs to be protected and not private?

Reply:
They both protect methods within from being called. The difference being that methods in private modules cannot be called by explicit recipients. Also, my previous controllers always made use of `protected`.